app/controllers/notification_settings_controller.rb

```diff
@@ -19,13 +19,26 @@ def create
 
     authorize notification_setting
 
-    result = if notification_setting.save
-               { notice: 'The setting was successfully saved.' }
-             else
-               { error: "The setting could not be saved: #{notification_setting.errors.full_messages.join(', ')}" }
-             end
-
-    redirect_back fallback_location: '/rad_common/notification_settings', flash: result
+    respond_to do |format|
+      format.html do
```

Comment:
Is the html version still used? It can be removed if not.

Reply:
removed the html version